csep/utils/plots.py

```diff
@@ -1339,18 +1339,17 @@ def plot_comparison_test(results_t, results_w=None, axes=None, plot_args=None):
     ax.set_ylabel(ylabel)
     ax.set_title(title)
     ax.yaxis.grid()
-    xTickPos, _ = pyplot.xticks()
+    xTickPos = ax.get_xticks()
     ax.yaxis.set_major_locator(matplotlib.ticker.MaxNLocator(integer=True))
-    ax.set_ylim([ylims[0], ylims[1]])
-    ax.set_xlim([ax.get_xlim()[0] + 0.5, ax.get_xlim()[1] - 0.5])
+    ax.set_ylim([ylim[0], ylim[1]])
+    ax.set_xlim([ax.get_xlim()[0] - 0.5, ax.get_xlim()[1] + 0.5])
```

Comment:
This ax.set_xlim() change gives an extra edge border that I'm not sure is needed. This gives
![image](https://user-images.githubusercontent.com/69042881/126714952-069f5da5-28fc-46fa-92fd-625c1a239f97.png)

versus this, which was in the previous xlim definition
![image](https://user-images.githubusercontent.com/69042881/126715059-9b9e63cc-844a-4dcb-bd49-6eb3a496e6fb.png)

But this failed for low number of Models in the t-test. I'd suggest to leave it better like:

`ax.set_xlim([-0.5, len(results_t) - 0.5])`

It appears to work well with any number of forecasts

Reply:
that doesn't seem to work with only two models, it only displays one
<img width="1204" alt="image" src="https://user-images.githubusercontent.com/35315438/126718038-abf508e0-ab19-4b5d-9eb4-023129111b62.png">
if i use 
```
ax.set_xlim([-0.5, len(results_t) + 0.5]) 
```
it seems to work ok. am i missing something here?
<img width="828" alt="image" src="https://user-images.githubusercontent.com/35315438/126718086-dd06ceef-564b-496d-b6ec-360781a9ce2b.png">

Reply:
checking....

Reply:
i really like the way these plots look!

Reply:
Weird!  Using `ax.set_xlim([-0.5, len(results_t) - 0.5])` I got this plot correctly:
![image](https://user-images.githubusercontent.com/69042881/126719556-9cad5514-dad6-4313-b77c-60b55c87c704.png)

Reply:
i think i got it. i just pushed a new commit with that fix.